src/module.ts

```diff
@@ -148,6 +154,12 @@ declare module '@nuxt/schema' {
   }
   interface NuxtOptions {
     ['i18n']: UserNuxtI18nOptions
+    /**
+     * @internal
+     */
+    _i18n: {
```

Comment:
sorry, where is this option used? I couldn't find it. 😅

Reply:
Ah looks like I forgot to remove that, it's part of the original/previous implementation in https://github.com/BobbieGoede/i18n/pull/50, will clean it up